anonlink/entitymatch.py

```diff
@@ -3,31 +3,39 @@
 from anonlink._entitymatcher import ffi, lib
 
 import sys
+from operator import itemgetter
 
 from . import bloommatcher as bm
 from . import util
 
 log = logging.getLogger('anonlink.entitymatch')
 
 
-def python_filter_similarity(filters1, filters2):
+def python_filter_similarity(filters1, filters2, k, threshold):
     """Pure python method for determining Bloom Filter similarity
 
     Both arguments are 3-tuples - bitarray with bloom filter for record, index of record, bitcount
 
-    :return: A list of tuples *one* for each entity in filters1.
+    :return: A list of tuples *k* for each entity in filters1.
```

Comment:
I'm not against that proposal, but it would be nice to know what you think the advantage of passing `k=None` is over passing `k=len(whatever)`. Note that the latter is more explicit about what the intention is. Also note that, were we to allow `k=None`, the callee would require special code to check if `k==None` and, that being so, set `k=len(whatever)`, which is not obviously a net win to me.

In summary, I don't see a significant advantage to allowing `None`, and I think it obscures the intention a bit. (Note that I assume you're not proposing that `k=None` should be a default parameter. That could be a reasonable suggestion, but should happen over at #80.)